core/configurator.go

```diff
@@ -73,15 +73,12 @@ var ignoredDirectories = []string{
 	".git",
 }
 
-// GetLatestVersion checks if there's an updated EAP version supported by the CLI.
```

Comment:
Removed EAP function as it was not used for a while since we released this summer non-EAP linters. Will define the proper upgrade flow logic with the team later (not today).